openedx/core/djangoapps/coursetalk/models.py

```diff
@@ -0,0 +1,32 @@
+"""
+Models for CourseTalk configurations
+"""
+from __future__ import unicode_literals
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from config_models.models import ConfigurationModel
+
+
+class CourseTalkWidgetConfiguration(ConfigurationModel):
+    """
+    Enable CourseTalk widget on course page
+    """
+    platform_key = models.fields.CharField(
+        max_length=50,
+        help_text=_(
+            "The platform key associates CourseTalk widgets with your platform. "
+            "Generally, it is the domain name for your platform. For example, "
+            "if your platform is http://edx.org, the platform key is \"edx\"."
+        )
+    )
+
+    @classmethod
+    def get_platform_key(cls):
+        """
+        Returns platform_key if setting is enabled, else ''
```

Comment:
This docstring could be worded a bit better, in accordance with our coding standards: https://github.com/edx/edx-platform/wiki/Python-Guidelines#follow-pep-257